vmdk_plugin/plugin.go

```diff
@@ -361,6 +365,14 @@ func (d *vmdkDriver) Mount(r volume.MountRequest) volume.Response {
 		isReadOnly = true
 	}
 
+	_, exists := status["fstype"]
+
+	if !exists {
+		log.WithFields(
+			log.Fields{"name": r.Name, "error": "Invalid filesystem type."},
```

Comment:
can you use a var for message instead of copy-n-paste here and on line 374 ?  Also I believe the text ("failed mount - invalid filesystem type" ) should be the same in both cases, and ideally include some hint on what to do next. Though I have no idea what to do next ...